webhooks/webhookv1beta1/addresspool_webhook.go

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
I'd move this under internal/k8s/webhooks as it's not necessary to have this exported